agent/balrogagent/client.py

```diff
@@ -28,6 +28,14 @@ def get_url(api_root, path):
                 resp.raise_for_status()
                 data["csrf_token"] = resp.headers["X-CSRF-Token"]
 
+        # Workaround for https://bugzilla.mozilla.org/show_bug.cgi?id=1471109
+        # In our deployed environments, the Agent doesn't connect to admin over
```

Comment:
@mozbhearsum, for curiosity, in these environments, agent calls admin using ip address or DNS name?

Reply:
It's using localhost. We actually run into this check https://github.com/aio-libs/aiohttp/blob/c3c1e3a/aiohttp/cookiejar.py#L197 and never send the cookie back.

Reply:
Yeah, the key here is that it's not over https, so `unsafe` doesn't help us.